cmd/podman/utils.go

```diff
@@ -91,7 +91,7 @@ func startAttachCtr(ctr *libpod.Container, stdout, stderr, stdin *os.File, detac
 
 	err = <-attachChan
 	if err != nil {
-		return errors.Wrapf(err, "error attaching to container %s", ctr.ID())
+		return err
```

Comment:
just curious, do you not think it's helpful to have the ctr.id in the error message?  Might there be a case when it might be good to know or is it buried away already in the err?

Reply:
Should already be in any error returns from the API

Really, I just want to pass ErrDetach through unmodified, so if this turns out to be a problem, we can test for ErrDetach and pass that through without changes, and wrap the rest